Client/Frontend/Home/TwoLineCell.swift

```diff
@@ -4,6 +4,111 @@
 
 import UIKit
 
+class ThumbnailCell : UICollectionViewCell {
+    let textLabel = UILabel()
+    let imageView = UIImageView()
+    let margin = 10
+
+    override init() {
+        super.init()
+        setup()
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    private func setup() {
+        contentView.addSubview(textLabel)
+        contentView.addSubview(imageView)
+
+        textLabel.font = UIFont(name: "FiraSans-SemiBold", size: 13)
+        textLabel.textColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.blackColor() : UIColor.darkGrayColor()
+        textLabel.snp_makeConstraints({ make in
+            make.bottom.right.equalTo(self.contentView).offset(-self.margin)
+            make.left.equalTo(self.contentView).offset(self.margin)
+            make.height.equalTo(26)
+            return
+        })
+
+        imageView.layer.borderColor = UIColor.lightGrayColor().CGColor
+        imageView.layer.borderWidth = 1
+        imageView.snp_makeConstraints({ make in
+            make.top.equalTo(self.contentView).offset(self.margin)
+            make.left.equalTo(self.textLabel.snp_left)
+            make.right.equalTo(self.textLabel.snp_right)
+            make.bottom.equalTo(self.textLabel.snp_top)
+        })
+    }
+
+    required init(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+class TopSitesSeperator : UICollectionReusableView {
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        self.backgroundColor = UIColor.lightGrayColor()
+    }
+}
+
+class TopSitesRow : UICollectionViewCell {
```

Comment:
Why is this in `TwoLineCell.swift`? `TopSitesRow` is specific to top sites and not used anywhere else, so shouldn't it be a private class in `TopSitesPanel.swift`?